src/NameGenerator.elm

```diff
@@ -0,0 +1,116 @@
+module NameGenerator exposing
```

Comment:
This is a generator of proper names based on a given lexicon.
Can I find a better name for the module?